src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -25,6 +25,7 @@ internal static partial class OpenSsl
         // Special value of 0 means unlimited, -1 means the implementation (OpenSSL) default, which is currently 20 * 1024.
         private const string TlsCacheSizeCtxName = "System.Net.Security.TlsCacheSize";
         private const string TlsCacheSizeEnvironmentVariable = "DOTNET_SYSTEM_NET_SECURITY_TLSCACHESIZE";
+        internal const string OpenSslDebugEnvironmentVariable = "DOTNET_SYSTEM_NET_SECURITY_OPENSSL_MEMORY_DEBUG";
```

Comment:
If the tracking is limited to System.Net's usages (TLS), then OK.  If it's all memory allocations, SYSTEM_NET should not be in the name.

`DOTNET_OPENSSL_MEMORY_DEBUG` seems fine.